### PR TITLE
[DOCFIX] Fix transformation doc

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

File: docs/en/core-services/Transformation.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ There are two kinds of supported transformations:
 1. coalesce the files so that each file is at least a certain size and there will be a maximum of certain number of files.
 2. convert CSV files to Parquet files
 
-> In Alluxio version 2.3.0, the transformed data is always written in Parquet format.
+> In Alluxio version 2.5.0, the transformed data is always written in Parquet format.
```
Comment (gpang) [inline docs/en/core-services/Transformation.md:25]: I don't know from which version, but can we do something like `2.2 and later`?

**Review by gpang — COMMENTED** [review]
@bradyoo I left a minor comment.

File: docs/en/core-services/Transformation.md (hunk 1 above)
Comment (bradyoo) [reply docs/en/core-services/Transformation.md:25]: I don't know from which version so I'll just stick to 2.2.0 and later.

**Comment by bradyoo** [comment]
feedback addressed

**Review by gpang — APPROVED** [review]
LGTM

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**